### DISCUSSION TITLE
Support tab based indentation

### DISCUSSION THREAD

**Discussion by nishith83** [codex-team/editor.js#1947] [Ideas]
Hey folks,

We recently started playing around with editorjs and must say the capabilities are strong.
However found a basic improvement which should go in editor (comparing with Notion):

- The editor should support tab based indentation. As far as I have played around, don't see this on the fly. Think from a writer's usability, this should be added

If there's a way to do it (not talking checklists/lists), let me know :)

**Comment by slidebug** [comment]
Yes good idea, please support it 👍

**Comment by 7ammer** [comment]
I see that tabs are supported in bullet points. There is an open issue in the checklists repo for tab based indentation: https://github.com/editor-js/checklist/issues/33
